src/codeu/chat/DerbyStore.java

```diff
@@ -137,7 +204,7 @@ public void updateLastMessage(Message m) throws SQLException {
 				// Add each created user to the store object
 				allUsers.insert(userid, user);
 			}
-			System.out.println("Successfully retrieved users");
+			LOG.info("Accessed users table.");
 		}
 		catch (Exception ex) {
 			ex.printStackTrace();
```

Comment:
I don't think I'm expecting any exceptions for my code. The only part where I was doing this was when creating the tables, but I changed it so that the constructor doesn't use exception control flow.